buildConfiguration.xml

```diff
@@ -15,6 +15,7 @@
       <project name="Microsoft.IdentityModel.Protocols" />
       <project name="Microsoft.IdentityModel.Protocols.WsFederation" />
       <project name="Microsoft.IdentityModel.Protocols.OpenIdConnect" />
+	   <project name="Microsoft.IdentityModel.Tokens.Extensions" />
     </src>
```

Comment:
Looks like the spacing here is a bit off...this is something I must have overlooked when I added this file in.

Reply:
Fixed: replaced a tab with whitespace.